From 75840ad8b40068b70792a27f1fdc7cccf3d28f7c Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@theqtcompany.com>
Date: Wed, 16 Sep 2015 13:30:38 +0300
Subject: [PATCH] qdoc: workaround for gcc's ICE

Remove c++11 option from compilation to prevent gcc's internal compiler error:
in add_stores, at var-tracking.c:6000

Signed-off-by: Samuli Piippo <samuli.piippo@theqtcompany.com>
---
 src/tools/qdoc/qdoc.pro | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/tools/qdoc/qdoc.pro b/src/tools/qdoc/qdoc.pro
index f0df113..a847482 100644
--- a/src/tools/qdoc/qdoc.pro
+++ b/src/tools/qdoc/qdoc.pro
@@ -4,6 +4,8 @@
     requires(!contains(QT_DISABLED_FEATURES, dom))
 }
 
+QMAKE_CXXFLAGS_CXX11 =
+
 option(host_build)
 QT = core xml
 
-- 
1.9.1

